### ISSUE TITLE
PHP 8 - Undefined constant "Eolica\NovaLocaleSwitcher\parent"

### ISSUE THREAD

**Issue by powellblyth** [Eolica-Web/nova-locale-switcher#2]
When running the tool using PHP 8, I get the following error

```
Error
Cannot call constructor
http://localhost:6080/es/admin/dashboards/main
```
```
Eolica\NovaLocaleSwitcher\LocaleSwitcher::__construct
vendor/eolica/nova-locale-switcher/src/LocaleSwitcher.php:17
```
This is because `Eolica\NovaLocaleSwitcher\LocaleSwitcher` calls `parent::__construct`  but this method does not exists since Laravel\Nova\Tool is abstract and has never had a constructor.

The simple fix for this is to remove the call for parent::__construct

[EDIT corrected error message]

**Comment by powellblyth** [comment]
I've created this PR to remedy it.  #3

**Comment by powellblyth** [comment]
Issue not related to PHP8 was related to bug in laravel nova 3.19.0 fixed in 3.19.1
